libtransmission/utils.cc

```diff
@@ -359,15 +381,10 @@ std::string to_utf8(std::string_view sv)
             continue;
         }
 
-#ifdef ICONV_SECOND_ARGUMENT_IS_CONST
-        auto const* inbuf = std::data(sv);
-#else
-        auto* inbuf = const_cast<char*>(std::data(sv));
-#endif
         size_t inbytesleft = std::size(sv);
         char* out = std::data(buf);
         size_t outbytesleft = std::size(buf);
-        auto const rv = iconv(cd, &inbuf, &inbytesleft, &out, &outbytesleft);
+        auto const rv = iconv(cd, SecondIconvArg(std::data(sv)), &inbytesleft, &out, &outbytesleft);
```

Comment:
Amazin' :laughing: :tada:

I don't think I've ever seen that before as an ifdef avoidance technique